pandas/io/excel/_base.py

```diff
@@ -157,7 +157,7 @@
     Data type for data or columns. E.g. {{'a': np.float64, 'b': np.int32}}
     Use `object` to preserve data as stored in Excel and not interpret dtype.
     If converters are specified, they will be applied INSTEAD
-    of dtype conversion.
+    of dtype conversion.This will necessarily result in ``object`` dtype.
```

Comment:
Where this currently is, it appears to apply to the previous sentence about dtype conversion. It needs to be moved one sentence prior. Also, I think it'd be more clear do have it read:

> Use `object` to preserve data as stored in Excel and not interpret dtype, which will necessarily result in ``object`` dtype.

Reply:
Thanks a lot ! This is my first contribution so I am getting the hang of it !